src/main/js/field/LabeledField.js

```diff
@@ -21,7 +21,14 @@ class LabeledField extends Component {
 
         const baseClasses = `${labelClass} text-right`;
         const labelClasses = (required) ? `${baseClasses} required` : baseClasses;
-
+        let severity = 'ERROR';
+        let fieldMessage = '';
+        if (errorValue) {
+            severity = errorValue.severity;
+            fieldMessage = errorValue.fieldMessage;
```

Comment:
{ severity, fieldMessage } = errorValue;

Reply:
That was the original code I had.  However errorValue can be null because there are no errors.  When errorValue is null object destructuring fails.  So that's why we have the code that is there.  If we change errorValue to {} then the check for errorValue will always be true.  We would have to test that